x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/action/PutFilterActionResponseTests.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+package org.elasticsearch.xpack.core.ml.action;
+
+import org.elasticsearch.test.AbstractStreamableTestCase;
+import org.elasticsearch.xpack.core.ml.job.config.MlFilterTests;
+
+public class PutFilterActionResponseTests extends AbstractStreamableTestCase<PutFilterAction.Response> {
```

Comment:
Can this be a `AbstractStreamableXContentTestCase`? The toXContent method calls `filter.toXContent()` so you can use the filter parser

Reply:
Sure. I also did it for the `PutCalendarActionResponseTests`.